clang/test/Format/empty-macro.cpp

```diff
@@ -0,0 +1,5 @@
+// RUN: grep -Ev "// *[A-Z-]+:" %s | clang-format -style="{Macros: [A(x)=x]}" \
+// RUN:   | FileCheck -strict-whitespace %s
+
+// CHECK: A()
+A()
```

Comment:
Don't add a lit test. Instead, add a unittest similar to the  one you already added there.

Reply:
Sorry for misunderstood.